test/daemon/peerstore.spec.js

```diff
@@ -87,28 +89,29 @@ describe('peerstore features', () => {
       }
     }
 
-    const stream = client.send(request)
+    streamHandler.write(Request.encode(request))
 
-    const message = await stream.first()
+    const message = await streamHandler.read()
     const response = Response.decode(message)
     expect(response.type).to.eql(Response.Type.OK)
     expect(response.peerStore).to.eql({
       protos: [
-        '/mplex/6.7.0',
-        '/ipfs/id/1.0.0',
-        '/ipfs/ping/1.0.0',
         '/libp2p/circuit/relay/0.1.0',
+        '/ipfs/id/1.0.0',
+        '/ipfs/id/push/1.0.0',
+        // '/mplex/6.7.0', TODO: WHY?
```

Comment:
I added this comment because we were not getting `/mplex` here when I was starting this refactor. I  tried it now again and we still not get it. Any ideas?

Reply:
We used to advertise the muxers, but now it's just application protocols, which I believe is inline with how go behaves. The basic idea being, nothing in the connection upgrade process is advertised. That *may* eventually be in Multiaddrs, but for now is not. It's a separation of "this is how I talk to this peer", and "this is what features that peer supports".